cooked-validators/src/Cooked/Tx/Constraints/Optics.hs

```diff
@@ -0,0 +1,99 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+-- | Various Optics on 'TxSkels' and all the other types defined in
+-- 'Cooked.Tx.Constraints.Type'.
+module Cooked.Tx.Constraints.Optics where
```

Comment:
Indeed, 99 lines vs ~350 before the changes!

Since this module is driven largely by the needs of tweaks, shall it go to `Cooked.Attack.Tweak.Optics` or something?

More than OK to keep it as is though.